src/app/zap-templates/zcl/data-model/chip/wifi-network-diagnostics-cluster.xml

```diff
@@ -55,6 +72,18 @@ limitations under the License.
     <attribute side="server" code="0x0C" define="OVERRUN_COUNT" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="true">OverrunCount</attribute>
     <command source="client" code="0x00" name="ResetCounts" optional="true" cli="chip wifi_network_diagnostics resetcounts">
       <description>Reception of this command SHALL reset the Breacon and Packet related count attributes to 0</description>
-    </command>           
+    </command>
+    <event side="server" code="0x00" name="DisconnectionEvent" priority="info" optional="true">
+      <description>Indicate that a Node’s Wi-Fi connection has been disconnected as a result of de-authenticated or dis-association and indicates the reason.</description>
+      <field id="0" name="Disconnection" type="ENUM16"/>
```

Comment:
This is called "ReasonCode" in the spec, and the value is a uint16, not an enum16...

Reply:
I see this has been remove in https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/service_device_management/DiagnosticsWiFi.adoc

I found I am not able to get the latest changes from pdf, if I run 'make pdf' against tot, but it still show 
"Disconnection enum is derived from enum16"